Resources/Prototypes/_LostParadise/Maps/Pools/LostParadiseMapPool.yml

```
@@ -1,7 +1,8 @@
-- type: gameMapPool
-  id: LostParadiseMapPool
-  maps:
-  - LPPAsterisk
-  - LPPSubmarine
-  - LPPPebble
-  - LPPNormandy
+- type: gameMapPool
```

yamllint

[error] wrong new line character: expected \n

(new-lines)